Reqnroll.VisualStudio/Editor/Classification/DeveroomClassifications.cs

```diff
@@ -6,17 +6,17 @@ namespace Reqnroll.VisualStudio.Editor.Classification;
 
 internal static class DeveroomClassifications
 {
-    public const string Keyword = "deveroom.keyword";
-    public const string Tag = "deveroom.tag";
-    public const string Description = "deveroom.description";
-    public const string Comment = "deveroom.comment";
-    public const string DocString = "deveroom.doc_string";
-    public const string DataTable = "deveroom.data_table";
-    public const string DataTableHeader = "deveroom.data_table_header";
-
-    public const string UndefinedStep = "deveroom.undefined_step";
-    public const string StepParameter = "deveroom.step_parameter";
-    public const string ScenarioOutlinePlaceholder = "deveroom.scenario_outline_placeholder";
+    public const string Keyword = "reqnroll.keyword";
```

Comment:
Would you consider the use of GUIDs here instead? Speculating for a moment, what happens with the next generation of plug-in and the need that would arise of avoiding conflicts with these constants. If we still wanted to use the Reqnroll name, but need to distinguish these values - how would we do it?

Reply:
These are the "keys" the users need to use if they would like to reconfigure the coloring for themselves. I think this was even the key was displayed. As far as I see, now the display name is shown in the config editor. Maybe a good mix would be to keep the readable names, but suffix it with the package guid.